ewms_pilot/pilot.py

```diff
@@ -101,7 +105,21 @@ async def consume_and_reply(
         timeout=timeout_outgoing,
     )
 
+    housekeeper = Housekeeping()
+    staging_dir = debug_dir if debug_dir else Path("./tmp")
+
     try:
+        # Init command
+        if init_cmd:
+            # TODO - call func that makes subprocess shell using asyncio-tasks w/ housekeeper.basic_housekeeping()
+            await run_init_command(
+                init_cmd,
+                init_timeout,
+                housekeeper,
+                staging_dir,
+            )
```

Comment:
Would it be better to run the init cmd before opening the queues?  That way we don't need to do "housekeeping" and if there is a node error we don't even connect to the queues.

Reply:
That's a good idea. As for housekeeping, I separated queue-related housekeeping from "basic" housekeeping, although there's not much going on in "basic" housekeeping currently

Reply:
We could put htchirp stuff in basic housekeeping

Reply:
That's an interesting idea